ui/recieve_page.go

```diff
@@ -1 +1,301 @@
 package ui
+
+import (
+	"time"
+
+	"gioui.org/layout"
+	"gioui.org/op/paint"
+	"gioui.org/unit"
+
+	"github.com/atotto/clipboard"
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/raedahgroup/godcr-gio/ui/decredmaterial"
+	"github.com/raedahgroup/godcr-gio/wallet"
+	"github.com/skip2/go-qrcode"
+)
+
+var (
+	listContainer = &layout.List{Axis: layout.Vertical}
+)
+
+func (win *Window) Receive() {
+	if win.walletInfo.LoadedWallets != 0 {
+		win.setDefaultPageValues()
+	}
+
+	body := func() {
+		layout.Stack{}.Layout(win.gtx,
+			layout.Expanded(func() {
+				win.ReceivePageContents()
+			}),
+		)
+	}
+	win.TabbedPage(body)
+}
+
+func (win *Window) ReceivePageContents() {
+	ReceivePageContent := []func(){
+		func() {
+			win.pageFirstColumn()
+		},
+		func() {
+			layout.Center.Layout(win.gtx, func() {
+				if win.outputs.err.Text != "" {
+					win.outputs.err.Layout(win.gtx)
+				}
+			})
+		},
+		func() {
+			win.selectedAccountColumn()
+		},
+		func() {
+			win.qrCodeAddressColumn()
+		},
+		func() {
+			layout.Flex{}.Layout(win.gtx,
+				layout.Flexed(.35, func() {
+				}),
+				layout.Flexed(1, func() {
+					if win.addressCopiedLabel.Text != "" {
+						win.addressCopiedLabel.Layout(win.gtx)
+					}
+				}),
+			)
+		},
+	}
+
+	listContainer.Layout(win.gtx, len(ReceivePageContent), func(i int) {
+		layout.Inset{Left: unit.Dp(20)}.Layout(win.gtx, ReceivePageContent[i])
+	})
+	if win.isNewAddrModal {
+		win.drawMoreModal()
+	}
+	if win.isInfoBtnModal {
+		win.drawInfoModal()
+	}
+}
+
+func (win *Window) pageFirstColumn() {
+	layout.Flex{}.Layout(win.gtx,
+		layout.Flexed(.6, func() {
+			win.outputs.receivePageTitle.Layout(win.gtx)
+		}),
+		layout.Flexed(.4, func() {
+			layout.Inset{Top: unit.Dp(4), Bottom: unit.Dp(20)}.Layout(win.gtx, func() {
+				layout.Flex{}.Layout(win.gtx,
+					layout.Flexed(.5, func() {
+						if win.inputs.receiveIcons.info.Clicked(win.gtx) {
+							win.isInfoBtnModal = true
+							win.isNewAddrModal = false
+						}
+						win.outputs.info.Layout(win.gtx, &win.inputs.receiveIcons.info)
+					}),
+					layout.Flexed(.5, func() {
+						for win.inputs.receiveIcons.more.Clicked(win.gtx) {
+							if win.isNewAddrModal {
+								win.isInfoBtnModal = false
+								win.isNewAddrModal = false
+							} else {
+								win.isNewAddrModal = true
+							}
+						}
+						win.outputs.more.Layout(win.gtx, &win.inputs.receiveIcons.more)
+					}),
+				)
+			})
+		}),
+	)
+}
+
+func (win *Window) selectedAccountColumn() {
+	layout.Flex{}.Layout(win.gtx,
+		layout.Flexed(0.22, func() {
+		}),
```

Comment:
Use the `win.combined.sel` widget. Should probably rename it to `accoutSel` first though

Reply:
I dont follow.

Reply:
There is a `decredmaterial.Select` widget that in window (`win.combined.sel`). You can see how it looks and works on the send page. It allows showing and switching accounts based on the currently selected wallet. Also the toRecieve icon should be on the transaction page beside the toSend icon (for now).

Reply:
just to be sure, this is not the account selector function. this is just to display the selected account information. and is is shabbily implemented on the wallet page.